doc/00-onboarding-guide.md

```diff
@@ -1,30 +1,31 @@
-# Onboarding process for using Azure Cli Codegen (autorest.az)
+# Onboarding Process for Using Azure Cli Code-gen (autorest.az)
```

Comment:
there is another file called "onboarding-guide.md" at the same folder. please clear that folder and link to this one.

Reply:
remove the other one

---
In reply to: [449503255](https://github.com/Azure/autorest.az/pull/461#discussion_r449503255) [](ancestors = 449503255)